.github/workflows/draft-new-release.yml

```diff
@@ -0,0 +1,42 @@
+name: Draft new release
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
```

Comment:
We added this because it's also specified in the example: https://github.com/release-drafter/release-drafter#usage 🚀 

Might be safer to keep it, I'd say